### DISCUSSION TITLE
Getting the Auth.User in Python Backend using Oauth

### DISCUSSION THREAD

**Discussion by cfortuner** [supabase/supabase#15309] [Questions]
Hi!

I'm wondering, what's the right way to get a auth.user and subsequently, auth.identities in my python backend?

For context, I am trying to understand, how to create a user using a Github app installation flow.

In this flow, a backend route handles the github installation payload which provides github identity / auth information to my backend.

I'd like to be able to create my user when this data is received, but I don't know how to do this in python.

Can someone share on how this could be done?

TLDR
how do you sign up a user on your backend using github identty?
how do you get auth.users and auth.identities information in your python backend?
